elasticsearch/src/main/scala/akka/stream/alpakka/elasticsearch/scaladsl/ElasticsearchFlow.scala

```diff
@@ -160,15 +158,62 @@ object ElasticsearchFlow {
                                     typeName: String,
                                     settings: ElasticsearchWriteSettings,
                                     elasticsearchClient: RestClient,
-                                    writer: MessageWriter[T]) =
-    Flow
-      .fromGraph(
-        new impl.ElasticsearchSimpleFlowStage[T, C](indexName, typeName, elasticsearchClient, settings, writer)
+                                    writer: MessageWriter[T]) = {
+    val (minBackoff, maxBackoff, maxRetries) = settings.retryLogic match {
+      case RetryNever => (Duration.Zero, Duration.Zero, 0)
```

Comment:
This could return a different flow for `RetryNever` which would just use the `ElasticsearchSimpleFlowStage` almost as before.

Reply:
I've integrated this suggestion, but I'm not sure it is clearer than always using the RetryFlow with a 0 max retries